gitpod-io/observability
# Delete unused jsonnet code

Fixes #409 

---

Monitoring-central is completely written in jsonnet today, but most of monitoring-satellite is managed with our Go installer. We still use the `import` fields to import some raw YAML manifests though, which are generated in this same repository using jsonnet. Examples:
* CRDs, using Jsonnet and Jsonnet Bundler to keep them updated with upstream.
* Grafana, using Jsonnet to transform dashboard JSONs into configmaps
* HTTP-prober, used by preview environments to establish one of their SLOs.


I've removed all code that is not touched by the examples above.

#### How to test

After deleting the code, I can run `make generate` and make sure that the generated manifests are still the exact same as before
